src/test/java/unit/Group.java

```diff
@@ -293,6 +296,39 @@ private static void expectListedMemberWithPolicies(
         "test user has the right policies");
   }
 
+  private TestCommand.Result expectGroupListedMemberWithPoliciesTableFormat(
+      String name, String email, GroupPolicy... policies) throws JsonProcessingException {
+    // call `terra group list` in table format
+    TestCommand.Result cmd = TestCommand.runCommand("group", "list");
+
+    // assert header is correct
+    String[] rows = cmd.stdOut.split("\\n");
+    String[] rowHead = rows[0].split("\\s+");
+    assertEquals("NAME", rowHead[0]);
+    assertEquals("EMAIL", rowHead[1]);
+    assertEquals("MEMBERS", rowHead[2]);
+    assertEquals("POLICIES", rowHead[3]);
+
+    // assert name, email and policies are correct
+    boolean nameAndEmailExists = false;
+
+    System.out.println(name);
+    System.out.println(email);
+    for (int i = 1; i < rows.length; i++) {
+      System.out.println(rows[i]);
+      // Convert to lower-case because email in broad.json is mixed case
+      if (rows[i].contains(email) && rows[i].contains(name)) {
+        nameAndEmailExists = true;
+        for (var policy : policies) {
+          assertThat("Row has all policies.", rows[i].contains(policy.toString()));
+        }
+      }
+    }
+    // name and email address for group member should be included in table output.
```

Comment:
put this comment inside the `assertTrue()` statement so it shows up when the assertion isn't met.